config/phenomenal_tools2container.yaml

```diff
@@ -325,7 +325,7 @@ assignment:
 - docker_image_override: mtblisa
   docker_owner_override: phnmnl
   docker_repo_override: container-registry.phenomenal-h2020.eu
-  docker_tag_override: v0.10.0_cv0.7.10.98
+  docker_tag_override: latest
```

Comment:
We need to fix these container versions!

Reply:
@ilveroluca I hadn't yet asked for a review yet, did I!? ;)

Reply:
oops :joy:

Reply:
I'll fix the version when I'm sure it's ready! (soon)